src/mongocrypt-ciphertext.c

```diff
@@ -20,13 +20,17 @@
 void
 _mongocrypt_ciphertext_init (_mongocrypt_ciphertext_t *ciphertext)
 {
+   BSON_ASSERT_PARAM (ciphertext);
+
    memset (ciphertext, 0, sizeof (*ciphertext));
 }
 
 
 void
 _mongocrypt_ciphertext_cleanup (_mongocrypt_ciphertext_t *ciphertext)
 {
+   BSON_ASSERT_PARAM (ciphertext);
```

Comment:
Suggest `if (!ciphertext) { return; }` for consistency with other cleanup functions.

Reply:
:+1: Change coming in next commit.